portal-api/src/createApp.js

```diff
@@ -23,12 +23,13 @@ const { authRouter, openRouter, authRouterAllowXss } = require('./v1/routes');
 configurePassport(passport);
 
 const app = express();
-// TODO: re-enable Helmet (Jira - 4998)
-// app.use(
```

Comment:
we need to re-enable this. can we add it back, but leave it disabled for now?

Reply:
Reverted it back